lib/view_model/wallet_list/wallet_list_view_model.dart

```diff
@@ -55,4 +55,8 @@ abstract class WalletListViewModelBase with Store {
             info.type == _appStore.wallet!.type,
         isEnabled: availableWalletTypes.contains(info.type))));
   }
+
+  bool checkIfAuthRequired(){
+    return getIt.get<AuthService>().requireAuth();
```

Comment:
Make AuthService a dependency in this view model instead of directly accessing it from getIt